test/tests/dialog-modal_datepicker.js

```diff
@@ -696,7 +706,7 @@ ariaTest('SPACE on cancel button does not select date', exampleFile, 'okay-cance
 
   await setDateToJanFirst2019(t);
   await t.context.session.findElement(By.css(ex.buttonSelector)).click();
-  await t.context.session.findElement(By.css(ex.currentlyFocusedButton)).sendKeys(Key.ARROW_RIGHT);
+  await t.context.session.findElement(By.css(ex.currentlyFocusedDay)).sendKeys(Key.ARROW_RIGHT);
   await t.context.session.findElement(By.css(ex.cancelButton)).sendKeys(Key.SPACE);
```

Comment:
Should all `Key.SPACE`s be replaced with `' '`? Maybe we can lint it also?

Reply:
Sounds like a good idea.

Reply:
@zcorpan 

I just did some testing and we can only do this when an keyboard event handler is associated with a button.  So I think we should only use `" " ` technique when needed and use `Key.SPACE` when it works correctly,